lib/main_containers.pm

```diff
@@ -127,8 +127,11 @@ sub load_host_tests_containerd_nerdctl {
 }
 
 sub load_host_tests_helm() {
-    if (is_tumbleweed || is_leap || is_sle) {
-        loadtest "containers/helm";
+    if (is_sle('15-sp4+')) {
+        loadtest "containers/helm_eks";
+        loadtest "containers/helm_k3s";
+    } elsif (is_opensuse) {
+        loadtest "containers/helm_k3s";
     }
```

Comment:
I am thinking now of having a `else` section with `die` message with `Helm not supported on this host` or `Helm can't be installed on this host` or something similar, wdyt?

Reply:
I thought the same if not is weird behavior

Reply:
done